ext/reflection/php_reflection.c

```diff
@@ -6661,28 +6655,22 @@ ZEND_METHOD(ReflectionProperty, isFinal)
 ZEND_METHOD(ReflectionExtension, __construct)
 {
 	zval *object;
-	char *lcname;
 	reflection_object *intern;
 	zend_module_entry *module;
 	char *name_str;
 	size_t name_len;
-	ALLOCA_FLAG(use_heap)
 
 	if (zend_parse_parameters(ZEND_NUM_ARGS(), "s", &name_str, &name_len) == FAILURE) {
 		RETURN_THROWS();
 	}
 
 	object = ZEND_THIS;
 	intern = Z_REFLECTION_P(object);
-	lcname = do_alloca(name_len + 1, use_heap);
-	zend_str_tolower_copy(lcname, name_str, name_len);
-	if ((module = zend_hash_str_find_ptr(&module_registry, lcname, name_len)) == NULL) {
-		free_alloca(lcname, use_heap);
+	if ((module = zend_hash_str_find_ptr_lc(&module_registry, name_str, name_len)) == NULL) {
```

Comment:
I wonder if it makes sense as a follow-up to change the ZPP specifier to return a zend_string.